packages/network/lib/agent.ts

```diff
@@ -148,17 +148,17 @@ export const regenerateRequestHead = (req: http.ClientRequest) => {
   }
 }
 
-const getFirstWorkingFamily = (
-  { port, host }: http.RequestOptions,
+export const getFirstWorkingFamily = (
+  { port, host }: Pick<http.RequestOptions, 'port' | 'host'>,
   familyCache: FamilyCache,
-  cb: Function,
+  cb: (family?: net.family) => void,
 ) => {
   // this is a workaround for localhost (and potentially others) having invalid
   // A records but valid AAAA records. here, we just cache the family of the first
   // returned A/AAAA record for a host that we can establish a connection to.
   // https://github.com/cypress-io/cypress/issues/112
 
-  const isIP = net.isIP(host)
+  const isIP = net.isIP(host) as net.family
```

Comment:
Reading the docs for `net.isIP` this can return 4 | 6 | 0. `net.family` is just 4 | 6. I think this provides clarity and explains why we have the check on 163.

```suggestion
  const isIP = net.isIP(host) as net.family | 0
```